tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/SimpleQueryExpressionWithDefinedTypeTest.java

```diff
@@ -363,6 +363,13 @@ public void testQueryWithRecordVarInLetClause() {
                 "deptAccess:\"XYZ\", address:{city:\"Colombo\", country:\"SL\"}}");
     }
 
+    @Test(description = "Test looping a stream with foreach")
+    public void testForeachStream() {
+        BValue[] returnValues = BRunUtil.invoke(result, "testForeachStream");
+        Assert.assertNotNull(returnValues);
+        Assert.assertTrue(((BBoolean) returnValues[0]).booleanValue());
```

Comment:
We can assert this in b7a itself, right? We can maybe change this also separately.

Reply:
This was done since the rest of the cases use the same approach.